Gemfile.lock

```diff
@@ -337,18 +377,26 @@ DEPENDENCIES
   oauth (~> 0.3.6)
   oily_png (~> 1.1.1)
   parallel (~> 1.1)
-  poltergeist (~> 1.6)
+  parallel_tests (~> 1.9.0)
+  poltergeist (~> 1.7.0)
   pry
   pry-byebug
   rake (~> 10.4)
   rb-fchange
   rb-fsevent
   rb-inotify
+  require_all (~> 1.3.2)
   rspec (~> 3.3)
+  rubocop (~> 0.34.2)
   ruby-duration (~> 3.1)
   sass (~> 3.4, < 3.4.6)
+  selenium-webdriver (~> 2.48.0)
   signet (~> 0.6)
+  site_prism (~> 2.7)
   slim (~> 3.0)
   therubyracer
   uglifier (~> 2.0.1)
   wraith (~> 1.3.0)
+
```

Comment:
You'll need to remove these last three lines. It causes a headache with the builds. They are coming back for me too. I'll have to have a look into it when Paul's back.